CHANGELOG.md

```diff
@@ -1,15 +1,20 @@
 # splat Release Notes
 
-### 0.32.4
+### 0.33.0
 
-- Fix splat on Windows not using forward slashes on generated paths.
-- Setup CI to be run on Windows and Macos too.
-- Fix `asmtu` segments not writing the other corresponding sections into the generated assembly file.
+* Fix splat on Windows not using forward slashes on generated paths.
+* Setup CI to be run on Windows and Macos too.
+* Fix `asmtu` segments not writing the other corresponding sections into the generated assembly file.
+* New yaml option: `keep_complete_c_asm_splits_for_objdiff_compatibility`
```

Comment:
`make_full_asm_splits` ?

Reply:
`make_asm_generated_again`, you can make red hats and everything

Reply:
Naming stuff is tricky.

It's creating an asm file for each c segment
- `create_objdiff_asm`
- `always_create_asm`
- `create_asm_for_c_segments`
- `always_split_asm`

Reply:
I'm not a fan of these... discounting the first one obviously :P

`always_create_asm` implies this will always create asm, which isn't true if you have that mode disabled. same with `always_split_asm`. also neither of those indicate that we're making one asm file for the whole segment. 

`create_asm_for_c_segments` is what splat does in the first place. it's just not creating one big asm file for c segments but rather several small ones.

`create_objdiff_asm` I think is the best of those 4, but I'd hope we can avoid calling this "the objdiff feature" since it theoretically could have other uses and isn't just a kludge for objdiff

you're not a fan of `make_full_asm_splits`?

`create_full_asm_blobs`?
`make_full_asm_blobs`?

Reply:
I was just throwing a few more names out there, I wasn't particularly convinced by any of mine either 😅 nothing has quite "clicked" as being a succinct explanation of what the feature is.. 

is `complete` better than `full`?

Is it creating one asm *per* c segment, or is it 1 enormous asm blob that includes all code?

Reply:
Something like `make_full_asm_splits` makes me think it is a feature for  `asm` segments only, when this only applies to `c` and `cpp` segments.

But I can't think on a better name that includes the `c` segment. Something like `full_s_file_for_c_seg` sounds terrible

Reply:
'make_full_disasm_for_cod(e)'

Reply:
how about `make_full_object_disasm` ?

Reply:
`make_full_object_disasm` sounds like all the sections will be emitted into the same assembly file (like disassembling a single TU instead of per section) like `asmtu` does.

I guess `make_full_disasm_for_code` will do